docs/samples/python/presidio_notebook.ipynb

```diff
@@ -152,9 +156,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "converted_analyzer_results = [result.to_dict() for result in analyzer_results]\n",
+    "new_results = [AnalyzerResult(result.entity_type, result.start, result.end, result.score) for result in analyzer_results]\n",
     "\n",
-    "print(converted_analyzer_results)\n"
+    "[print(result) for result in new_results]"
```

Comment:
print(new_results) won't give the same output?

Reply:
no, it prints the objects